pkgs/top-level/python-packages.nix

```diff
@@ -1208,6 +1208,21 @@ in modules // {
     };
   });
 
+  attrs = buildPythonPackage {
+    name = "attrs-15.2.0";
+    src = pkgs.fetchurl {
+      url =
+      "https://pypi.python.org/packages/8b/76/c57eefda827b981135ccacd4328fceaa3693f79d9da1e5d78fbe59ebd0c4/attrs-15.2.0.tar.gz";
```

Comment:
can you use the pypi mirror with the old url scheme here and also below. It should work (but occasionally doesn't).